### PR TITLE
Fixed hostStats problems

### PR DESCRIPTION
**Pull request by pablomendezroyo**
**Change disk commmand**

`df / --block-size=1`   --->   ` df / -B 1`

 **Solved spelling mistake in HostStats.**

Switch disk and memory

### CODE REVIEW

**Comment by dapplion** [comment]
Thanks! Please install this version on your DAppNode and make sure the stats show as expected. Attach screen capture of the dashboard view and make sure the commit is correct

**Comment by pablomendezroyo** [comment]
> Thanks! Please install this version on your DAppNode and make sure the stats show as expected. Attach screen capture of the dashboard view and make sure the commit is correct

DNP_DAPPMANAGER version successfully installed and tested in my dappnode.

Screenshot from 2020-11-17 13-46-34

Screenshot from 2020-11-17 13-47-11
Screenshot from 2020-11-17 13-46-50

**Comment by dapplion** [comment]
Note that RAM is shown in MB when it should be GB. I think on this distro, the --bytes flag is ignored by free.

**Comment by pablomendezroyo** [comment]
Before continue with the tests modification, I just want to make sure If you are agree with the changes.

**Migration from linux commands to node-os-utils library**

Screenshot from 2020-11-17 23-46-07

File: packages/dappmanager/src/utils/toBytes.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ToBytes } from "../types";
+
+/**
+ * Function reuturns bytes. Accepts mb, kb and gb
+ * @param param0
+ */
+export function toBytes({
```
Comment (dapplion) [inline packages/dappmanager/src/utils/toBytes.ts:7]: You can use as signature
```ts
toBytes(amount: number, unit: "kb" | "mb" | "gb")
```
Not sure it's necessary to nest the options and the amount and unit types do not overlap so it's "safe" to use normal args

**Review by dapplion — CHANGES_REQUESTED** [review]

File: packages/dappmanager/src/utils/toBytes.ts (hunk 1 above)
Comment (pablomendezroyo) [reply packages/dappmanager/src/utils/toBytes.ts]: done

File: packages/admin-ui/src/common/types.ts [hunk 2]
```diff
@@ -1061,36 +1061,34 @@ export interface SystemInfo {
...
  * Host machine CPU used
  */
 export interface HostStatCpu {
-  usedFraction: number;
+  usedPercentage: number;
+}
+/**
+ * Types of conversions accepted by toBytes() function
+ */
+export interface ToBytes {
```
[the remark this answers is not in the document]
Comment (pablomendezroyo) [reply packages/admin-ui/src/common/types.ts]: deleted interface

**Review by dapplion — APPROVED** [review]
LGTM

**Pull request merged**